lib/timeline/component/GraphGroup.js

```diff
@@ -50,35 +55,37 @@ GraphGroup.prototype.setItems = function(items) {
   }
 };
 
+GraphGroup.prototype.getItems = function () {
```

Comment:
does this need a getter? can't we just access the itemsData directly? Why the abstraction?